yarn-project/pxe/src/database/database.ts

```diff
@@ -24,6 +24,18 @@ export interface Database extends ContractDatabase {
    */
   getAuthWitness(messageHash: Fr): Promise<Fr[]>;
 
+  /**
+   * Adding a capsule to the capsule dispenser.
```

Comment:
```suggestion
   * Adding a capsule to the capsule dispenser.
   * @remarks A capsule is a "blob" of data that is passed to the contract through an oracle.
```

Everybody would be searching what a capsule is.